cfs-require-import.php

```diff
@@ -8,32 +8,50 @@
 Author URI: https://www.vanpattenmedia.com/
 */
 
-class CfsRequireImport {
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
 
+class CfsRequireImport {
+	
 	function __construct() {
-		register_activation_hook( plugin_basename( __FILE__ ), array( $this, 'import_cfs_fields' ) );
+		
+		/* Register activation hook. */
+		register_activation_hook( plugin_basename( __FILE__ ), array( $this , 'import_cfs_fields' ) );
+		
+		/* Add admin notice */
+		add_action( 'admin_notices' , array( $this , 'print_import_notice' ) );
 	}
-
+	
+ 	/* Runs only when the plugin is activated. */
 	public static function import_cfs_fields() {
 		$fields = file_get_contents( trailingslashit( dirname( __FILE__ ) ) . 'fields.json' );
 
 		$options = array(
-			'import_code' => json_decode( stripslashes( $fields ), true ),
+			'import_code' => json_decode( $fields , true ),
```

Comment:
Very small style thing, but you can remove the space between `$fields` and the argument separating comma (e.g. `$fields ,` becomes `$fields,`)

Reply:
Ok.